dotCMS/src/main/webapp/dotcms-webcomponents/dotcms-webcomponents.js

```diff
@@ -0,0 +1,896 @@
use strict";var e=n(0),o=n(32).trim;e({target:"String",proto:!0,forced:n(74)("trim")},{trim:function(){return o(this)}})},function(t,r,n){"use strict";var e=n(0),o=n(32).end,i=n(74)("trimEnd"),u=i?function(){return o(this)}:"".trimEnd;e({target:"String",proto:!0,forced:i},{trimEnd:u,trimRight:u})},function(t,r,n){"use strict";var e=n(0),o=n(32).start,i=n(74)("trimStart"),u=i?function(){return o(this)}:"".trimStart;e({target:"String",proto:!0,forced:i},{trimStart:u,trimLeft:u})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("anchor")},{anchor:function(t){return o(this,"a","name",t)}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("big")},{big:function(){return o(this,"big","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("blink")},{blink:function(){return o(this,"blink","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("bold")},{bold:function(){return o(this,"b","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("fixed")},{fixed:function(){return o(this,"tt","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("fontcolor")},{fontcolor:function(t){return o(this,"font","color",t)}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("fontsize")},{fontsize:function(t){return o(this,"font","size",t)}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("italics")},{italics:function(){return o(this,"i","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("link")},{link:function(t){return o(this,"a","href",t)}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("small")},{small:function(){return o(this,"small","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("strike")},{strike:function(){return o(this,"strike","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("sub")},{sub:function(){return o(this,"sub","","")}})},function(t,r,n){"use strict";var e=n(0),o=n(12);e({target:"String",proto:!0,forced:n(13)("sup")},{sup:function(){return o(this,"sup","","")}})},function(t,r,n){"use strict";var e,o=n(4),i=n(52),u=n(23),a=n(49),c=n(103),s=n(2),f=n(17).enforce,l=n(78),h=!o.ActiveXObject&&"ActiveXObject"in o,p=Object.isExtensible,v=function(t){return function(){return t(this,arguments.length?arguments[0]:void 0)}},g=t.exports=a("WeakMap",v,c,!0,!0);if(l&&h){e=c.getConstructor(v,"WeakMap",!0),u.REQUIRED=!0;var d=g.prototype,y=d.delete,x=d.has,m=d.get,b=d.set;i(d,{delete:function(t){if(s(t)&&!p(t)){var r=f(this);return r.frozen||(r.frozen=new e),y.call(this,t)||r.frozen.delete(t)}return y.call(this,t)},has:function(t){if(s(t)&
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using for statements without curly braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5062311)